<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramPageJava</class>
 <widget class="QWidget" name="ProgramPageJava">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Java Version:</string>
       </property>
       <property name="buddy">
        <cstring>combo_version</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QComboBox" name="combo_version">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Fetching Versions...</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_main_class">
       <property name="text">
        <string>Main Class:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QComboBox" name="combo_start_mode">
       <item>
        <property name="text">
         <string>Main Class</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JAR File</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start Mode:</string>
       </property>
       <property name="buddy">
        <cstring>combo_start_mode</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_jar_file">
       <property name="text">
        <string>JAR File:</string>
       </property>
       <property name="buddy">
        <cstring>combo_jar_file</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="4">
      <widget class="QLabel" name="label_jar_file_help">
       <property name="text">
        <string>The specified Java archive has to contain the main() method and will be executed with the -jar option.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLabel" name="label_main_class_help">
       <property name="text">
        <string>The specified Java class has to contain the main() method.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QComboBox" name="combo_jar_file">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_class_path">
       <property name="text">
        <string>Class Path:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>list_class_path</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="4">
      <widget class="QListWidget" name="list_class_path"/>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="button_add_class_path_entry">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="button_remove_class_path_entry">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QPushButton" name="button_up_class_path_entry">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="4">
      <widget class="QLabel" name="label_class_path_help">
       <property name="text">
        <string>This list of class path entries will be passed to the Java virtual machine (JVM).</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="1" colspan="4">
      <widget class="QCheckBox" name="check_show_advanced_options">
       <property name="text">
        <string>Show Advanced Options</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_working_directory">
       <property name="text">
        <string>Working Directory:</string>
       </property>
       <property name="buddy">
        <cstring>combo_working_directory</cstring>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_options">
       <property name="text">
        <string>JVM Options:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>list_options</cstring>
       </property>
      </widget>
     </item>
     <item row="15" column="1" colspan="4">
      <widget class="QComboBox" name="combo_working_directory">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <item>
        <property name="text">
         <string>.</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="16" column="1" colspan="4">
      <widget class="QListWidget" name="list_options"/>
     </item>
     <item row="18" column="1">
      <widget class="QPushButton" name="button_add_option">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QPushButton" name="button_remove_option">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="18" column="3">
      <widget class="QPushButton" name="button_up_option">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item row="18" column="4">
      <widget class="QPushButton" name="button_down_option">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="button_down_class_path_entry">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1" colspan="4">
      <widget class="QLabel" name="label_options_help">
       <property name="text">
        <string>This list of options will be passed to the Java virtual machine (JVM).</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_jar_file_type">
       <property name="text">
        <string>JAR File Type:</string>
       </property>
       <property name="buddy">
        <cstring>combo_jar_file_type</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="4">
      <widget class="QComboBox" name="combo_jar_file_type">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>All Files</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JAR Files (*.jar)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QComboBox" name="combo_main_class">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="1" colspan="4">
      <widget class="QLabel" name="label_spacer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;spacer&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="5">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="4">
      <widget class="QCheckBox" name="check_show_class_path">
       <property name="text">
        <string>Show Class Path</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLabel" name="label_main_class_error">
       <property name="text">
        <string>&lt;main-class-error&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_version</tabstop>
  <tabstop>combo_start_mode</tabstop>
  <tabstop>combo_main_class</tabstop>
  <tabstop>combo_jar_file</tabstop>
  <tabstop>combo_jar_file_type</tabstop>
  <tabstop>check_show_class_path</tabstop>
  <tabstop>list_class_path</tabstop>
  <tabstop>button_add_class_path_entry</tabstop>
  <tabstop>button_remove_class_path_entry</tabstop>
  <tabstop>button_up_class_path_entry</tabstop>
  <tabstop>button_down_class_path_entry</tabstop>
  <tabstop>check_show_advanced_options</tabstop>
  <tabstop>combo_working_directory</tabstop>
  <tabstop>list_options</tabstop>
  <tabstop>button_add_option</tabstop>
  <tabstop>button_remove_option</tabstop>
  <tabstop>button_up_option</tabstop>
  <tabstop>button_down_option</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
